docs/UserGuide.md

```diff
@@ -65,16 +65,22 @@ you a one-stop interface to access a plethora of features to manage your finance
 ### Notes about naming convention
 - Cashflow refers to an income or expense.
 
+### Notes about program limitations
+- Maximum amount for each cashflow and total balance that the program can hold is 999,999,999,999.99
+- Minimum amount for each cashflow and total balance that the program can hold is -999,999,999,999.99
+- Total balance is separate from Income Balance and Expense Balance, where the latter do not have the same limitations.
```

Comment:
Sounds weird maybe 
Total Balance, Income balance, and Expense balance are different entities where the latter two do not have the same limitations